app/scripts/views/complete_reset_password.js

```diff
@@ -31,9 +32,15 @@ define(function (require, exports, module) {
     },
 
     events: {
+      'blur .check-password': 'onPasswordBlur',
```

Comment:
I just changed the mixin to be part of the password-strength-mixin and forgot to move this :) Thanks for the catch @vladikoff!